.github/workflows/thurloe-build-tag-publish.yml

```
@@ -88,6 +88,7 @@ jobs:
       id-token: 'write'
 
   set-version-in-dev:
+    if: ${{ github.ref_name == 'develop' }}
```

```suggestion
    if: ${{ github.event_name != 'pull_request' }}
```

I'd strongly prefer we act on the event type, not the ref. The ref can be develop from a pull request, in theory. We never want to deploy to dev from a pull request.

---

I'm deleting my comment in favor of this due to the potential edge case pointed out here. We definitely never want to deploy to dev from a PR, even if we deploy a PR to dev for testing.